python/lsst/scarlet/lite/component.py

```diff
@@ -237,6 +237,7 @@ def prox_spectrum(self, spectrum: np.ndarray) -> np.ndarray:
         """Apply a prox-like update to the spectrum"""
         # prevent divergent spectrum
         spectrum[spectrum < self.floor] = self.floor
+        spectrum[~np.isfinite(spectrum)] = 1e-20
```

Comment:
Not knowing this code-base particularly well, do you want to set this to `self.floor` as well, or does it make more sense to hard-code it as you have it here?

Reply:
Good point, yes I should do that.